casepro/utils/__init__.py

```diff
@@ -185,3 +185,23 @@ def get_language_name(iso_code):
         LANGUAGES_BY_CODE[iso_code] = lang
 
     return LANGUAGES_BY_CODE[iso_code]
+
+
+def humanise_seconds(seconds):
+    days, secs = divmod(seconds, 86500)
```

Comment:
I think that's ok. My main objection to minutes for storing times was accumulating inaccuracy. For display, it's unlikely anyone's averages will actually be less than 1 minute. 

The nice thing about Django's timesince is that it's localised, and at some point we're going to have to support proper localization of the UI.